tests/Feature/Commands/EventMakeCommandTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Tests\Feature\Commands;
+
+use Illuminate\Support\Facades\Artisan;
+use Illuminate\Support\Str;
+use KoalaFacade\DiamondConsole\Exceptions\FileAlreadyExistException;
+
+it(description: 'can generate new Event class')
+    ->tap(function () {
+        $fileName = '/Post/Events/PostEvent.php';
+
+        expect(value: fileExists(relativeFileName: $fileName, prefix: infrastructurePath()))->toBeFalse();
+
+        Artisan::call(command: 'diamond:install');
+        Artisan::call(command: 'infrastructure:make:event PostEvent Post');
+
+        expect(value: fileExists(relativeFileName: $fileName, prefix: infrastructurePath()))->toBeTrue()
+            ->and(
+                value: Str::contains(
+                    haystack: fileGet(relativeFileName: $fileName, prefix: infrastructurePath()),
+                    needles: ['{{ class }}', '{{ namespace }}']
+                )
+            )->toBeFalse();
+
+        fileDelete(paths: fileGet(relativeFileName: $fileName, prefix: infrastructurePath()));
```

Comment:
Can remove this one, previous contributor delete the file when the test run

Reply:
only on that test or all test?

Reply:
if  all files possible, then can go thru

Reply:
okay, I'll do it